assets/store_descriptions/fr-FR/strings.sh

```diff
@@ -1,13 +1,36 @@
 #X-Generator: crowdin.com
 # Strings for store descriptions. Sourced by generate_and_validate.sh
 
+intro='openHAB - "empowering the smart home" - une solution de domotique libre et indépendante de tout fournisseur ou technologie pour votre maison connectée'
 
+whatis='openHAB est une plateforme de domotique open source basée sur Java qui intègre et combine une vaste gamme de produits et technologies pour la maison connectée dans une seule et même solution.'
+rules='S'appuyant sur une couche d'abstraction unifiée, tous les dispositifs connectés sont exposés au moteur de règles d'automatisation centralisé et à différentes interfaces utilisateur.'
 
+supported='Produits pris en charge'
+bindings='Plus de 200 modules spécifiques prennent en charge différentes marques, dispositifs, technologies et protocoles, tels que Z-Wave, Philips Hue, Amazon Echo, Chromecast et Sonos. Découvrez tous les modules disponibles et les périphériques et fonctionnalités supportés sur https://docs.openhab.org/addons/bindings.html'
 
+home_automation_solutions='Solutions de domotique'
+lighting='Éclairage'
+heating='Chauffage'
+home_entertainment='Divertissement à domicile'
+security='Sécurité'
+open_protocols='Protocoles ouverts'
+special_useCases='Cas d'utilisation spéciaux'
 
+oss_community='Communauté open source'
 
+forum='openHAB, en tant qu'initiative open source, soutient vivement sa communauté très dynamique. Le forum avec ses plus de 13 000 membres est l'endroit où trouver conseils, aide et inspiration. Rejoignez la communauté openHAB et son forum sur https://community.openhab.org'
+report_issues='Si vous rencontrez un problème, veuillez nous en faire part sur le forum ou sur https://github.com/openhab/openhab-android/issues'
+translation='Si vous souhaitez traduire l'application dans votre langue, rejoignez-nous sur https://crowdin.com/project/openhab-android'
```

Comment:
@ghys Single quotes are causing issues with the validation script. I'll push a fix soon, please don't re-approve those strings for now.